### PR TITLE
[Windows] stop setuptools egg packaging.

### PR DESCRIPTION
**Pull request by seanyen**
This is the same treatment as #1068 but for Windows build.

This would need to be cherry-picked to `kinect-devel` too.

### CODE REVIEW

**Comment by seanyen** [comment]
@dirk-thomas This is ready for review and merge. Thanks.

**Comment by seanyen** [comment]
@dirk-thomas Friendly ping.

File: cmake/templates/python_distutils_install.bat.in [hunk 1]
```diff
@@ -16,4 +16,5 @@ set "CATKIN_BINARY_DIR=@CMAKE_BINARY_DIR@"
     build --build-base "@CMAKE_CURRENT_BINARY_DIR@" ^
     install %DESTDIR_ARG% @SETUPTOOLS_ARG_EXTRA@ ^
     --prefix="@SETUPTOOLS_INSTALL_PREFIX@" ^
-    --install-scripts="@SETUPTOOLS_INSTALL_PREFIX@\@CATKIN_GLOBAL_BIN_DESTINATION@"
+    --install-scripts="@SETUPTOOLS_INSTALL_PREFIX@\@CATKIN_GLOBAL_BIN_DESTINATION@" ^
+    --root="/"
```
Comment (dirk-thomas) [inline cmake/templates/python_distutils_install.bat.in]: I am surprised that it should be `/` on Windows?

Comment (seanyen) [reply cmake/templates/python_distutils_install.bat.in]: @dirk-thomas Both (`/` and `\`) work here but for the consistency I changed it to backslash.

Comment (dirk-thomas) [reply cmake/templates/python_distutils_install.bat.in]: Interesting - I wasn't even thinking slash vs. backslash. I was wondering what either does on Windows without a drive letter...

**Comment by dirk-thomas** [comment]
If it works for me that is good enough for me :wink: Thanks for the patch.

**Comment by dirk-thomas** [comment]
Cherry-picked to `kinetic-devel`: c6759139d35869c614bc739745b9d10297447a99.

**Pull request merged**